SIGS/app/assets/stylesheets/rooms_filter.scss

```diff
@@ -0,0 +1,27 @@
+.select-filter {
+    display: block;
+    width: 100%;
+    height: 34px;
+    padding: 6px 12px;
+    font-size: 14px;
+    line-height: 1.428571429;
+    color: #555555;
+    background-color: #fff;
+    background-image: none;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+    transition: border-color ease-in-out 0.15s, box-shadow ease-in-out 0.15s;
+}
+
+.form-control {
```

Comment:
This class already exist on:
![image](https://user-images.githubusercontent.com/18387694/56331029-054a7b00-6161-11e9-9d95-c051e2bc144f.png)

This way its better to not duplicate code. So replace `width: 100% !important;` on the another file and remove this class.